site/routes/definitions.js

```diff
@@ -56,6 +69,16 @@ export const routeDefinitions : Array<RouteDefinition> = [
     gen: state => state.events.map(({ startDateTime: { date, month, year }, slug }) => ({ date, month, year, slug })),
   },
   {
+    title: ({ category }) => 'Meet our team' + (category !== 'everyone' ? ` (${category})` : ''),
+    key: 'badgers',
+    route: 'about-us/people/{category?}/{page?}',
+    defaults: { category: 'everyone', page: 1 },
+    urlEncode: (_, key, val) => (val === 'ux & design' ? 'ux-design' : encodeURIComponent(val)),
+    urlDecode: (_, key, val) => (val === 'ux-design' ? 'ux & design' : decodeURIComponent(val)),
```

Comment:
Just chatted with @JoeStanton and we decided:
* Hard-code the list of categories in grapnql layer
* Each badger returns with a list of their categories. Each category has two properties name and slug
```
{
  firstName: ‘Alex’,
  ...
  categories: [
    {name: ‘Engineering’, slug: ‘engineering’},
    {name: ‘UX & Design’, slug: ‘ux-design’}
  ]
}
```
* The filtering of invalid categories should be done within graphql layer. If a badger has an invalid category assigned (not one of the hard-coded list) then this category shouldn’t be returned.